icu4c/source/data/brkitr/ja.txt

```diff
@@ -238,4 +238,940 @@ ja{
         "ーリンズ",
         "ーン",
     }
+    modelKeys {
```

Comment:
Since modelKeys and modelValues are both for your budouX model, could we introduce a common parent in the resource strucutre to hold them instead of placing them side by side in ja.txt and mixed with other thing?
I mean change it from 
```
ja {
  ...
  modelKeys {...}
  modelValues {...}
}
```
to something like 
```
ja {
  ...
  budoux_model{
     keys {...}
     values {...}
  }
}
```
but maybe some name other than "budoux_model"

Reply:
also, I would to see some documentation describe the structure of the key and value.  Similar to the combination of
1.  the "Model structure" section in https://github.com/unicode-org/lstm_word_segmentation#model-structure to explain what the model contains. AND
2.  the "Resource Source Format" section in 
https://docs.google.com/document/d/1EVK2CwOmUamJwMOMbbdTz7tuaV0IR21rMoH7a3pyFwE/edit#heading=h.4mbjqu1e77w which explain the structure of the resource

Reply:
1. No idea how to build this .res file. I have to change CLDR to separate this new file, don't I?
2. I'll create another PR for documentation.

Reply:
1. I've separate the model file to load dynamically
2. Add the BudouX doc: https://docs.google.com/document/d/18mQUSIpeaE70SvAbbRjDl8wPPbh1PnBjyzK-FytZjVA/edit?resourcekey=0-DIOL6sNoVSR6BfS_zVo6Jw#heading=h.jh587cvmr6n